src/mood.ts

```diff
@@ -141,6 +138,9 @@ export function meatMood(urKels = false, meat = baseMeat): Mood {
     }
   }
 
+  const flaskValue = usingPurse() ? 0.3 * baseMeat : 5;
```

Comment:
Is there a reason you moved these lines? It's NBD but I'm curious

Reply:
Honestly I haven't looked at this in a long time because I couldn't figure out why it was having the circular import issue.

This was one of the first attempts I had at a PR and I was even more ignorant than I am now so I feel like if I wanted to try to take a stab at this again I'd just do it all over

I'm guessing from my initial PR post I moved them because of this:

> It's possible when fully buffed up with effects the Half a Purse could still outperform whatever other offhand we're using instead with familiar weight because of the diminishing returns, but I'm unsure of how to account for this because I'm not sure when during the mood management this would be happening. I moved it to the end of mood.ts in a naïve attempt to account for this